Assets/_Application/Layers/LayerData.cs

```diff
@@ -146,38 +135,39 @@ public bool HasValidCredentials
         }
 
         [JsonIgnore] public bool HasProperties => LayerProperties.Count > 0;
-
-        [JsonIgnore] public Dictionary<string, LayerStyle> Styles => styles;
         
-        /// <summary>
-        /// Every layer has a default style, this is a style that applies to all objects and features in this
-        /// layer without any conditions.
-        /// </summary>
-        [JsonIgnore] public LayerStyle DefaultStyle => Styles[NameOfDefaultStyle];
+        [DataMember] protected string prefabId;
 
-        /// <summary>
-        /// Every layer has a default symbolizer, drawn from the default style, that can be queried for the appropriate
-        /// properties.
-        /// </summary>
-        [JsonIgnore] public Symbolizer DefaultSymbolizer => DefaultStyle.StylingRules[NameOfDefaultStyle].Symbolizer;
+        public string PrefabIdentifier //todo: this being settable is now very error sensitive. Will be refactored in ticket 3/4
+        {
+            get
+            {
+                return prefabId;
+            }
+            set
+            {
+                prefabId = value;
+                OnPrefabIdChanged.Invoke();
+            }
+        }
+        public UnityEvent OnPrefabIdChanged = new();        
 
         [JsonIgnore] public readonly UnityEvent<string> NameChanged = new();
         [JsonIgnore] public readonly UnityEvent<bool> LayerActiveInHierarchyChanged = new();
         [JsonIgnore] public readonly UnityEvent<Color> ColorChanged = new();
         [JsonIgnore] public readonly UnityEvent LayerDestroyed = new();
         [JsonIgnore] public readonly UnityEvent<int> LayerOrderChanged = new();
 
-        [JsonIgnore] public readonly UnityEvent<LayerData> LayerSelected = new();
-        [JsonIgnore] public readonly UnityEvent<LayerData> LayerDeselected = new();
+        [JsonIgnore] public readonly UnityEvent LayerSelected = new();
```

Comment:
Deze weghalen is onhandig - een listener moet kunnen weten welke layer selected is als die daarop wil acteren